contrib/catfish/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "catfish"
+pkgver = "4.18.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "intltool",
+    "python-build",
+    "python-distutils-extra",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+depends = [
+    "gdk-pixbuf",
+    "glib",
+    "gtk+3",
+    "pango",
+    "python",
+    "python-cairo",
+    "python-gobject",
+    "python-pexpect",
+    "virtual:cmd:locate!chimerautils-extra",
+    "xfconf",
+]
+pkgdesc = "Xfce file search tool"
+maintainer = "triallax <triallax@tutanota.com>"
+# TODO: https://gitlab.xfce.org/apps/catfish/-/issues/106
+license = "GPL-2.0-or-later"
+url = "https://xfce.org"
+source = f"$(XFCE_SITE)/apps/{pkgname}/{pkgver[:-2]}/{pkgname}-{pkgver}.tar.bz2"
```

Comment:
we generally avoid expanding pkgname in source url these days (ditto everywhere else)